server/settings.py

```diff
@@ -152,6 +152,10 @@ def env(variable, fallback_value=None):
         "task": "liveblog.blogs.tasks.remove_deleted_blogs",
         "schedule": crontab(hour="*/1"),  # run every hour
     },
+    "bandwidth:update": {
+        "task": "liveblog.bandwidth.tasks.fetch_bandwidth_usage",
+        "schedule": crontab(minute=0, hour=0),  # every midnight
```

Comment:
can we do this more often? If there is a busy liveblog instance, the limit might be consumed quickly and by the time we know they crossed the limit, their event could be over :)